tidy3d/plugins/mode/mode_solver.py

```diff
@@ -317,6 +328,9 @@ def data_raw(self) -> ModeSolverData:
         if self.mode_spec.group_index_step > 0:
             return self._get_data_with_group_index()
 
+        if self.mode_spec.bend_angle_rotation:
+            return self.rotated_mode_solver_data
```

Comment:
So this skips everything else that happens in the `data_raw` function. Thinking about it, I think that's fine for everything *except* potentially the normalization. I'm hoping that calling `self._normalize_modes()` on the `rotated_mode_solver_data`, so as to renormalize *after* the interpolation, will fix the TE00 issue. This can be added in the `rotated_mode_solver_data` property.